bin/cylc-view

```diff
@@ -117,7 +117,7 @@ def get_option_parser():
 
 @cli_function(get_option_parser)
 def main(parser, options, reg):
-    suite, suiterc = SuiteSrvFilesManager().parse_suite_arg(options, reg)
+    suite, suiterc = suite_srv_files_mgr.parse_suite_arg(options, reg)
```

Comment:
ditto

Reply:
(Obviously I realise at this point it might be easier to be consistent by changing the specific imports to non-specific, and that it doesn't really matter either way, but now that I've started this exercise in futility I have to ... be consistent :grimacing: )